apps/fp-frontend-app/src/app/components/Home.tsx

```diff
@@ -39,12 +40,13 @@ interface Props {
  */
 export const Home = ({ headerHeight, navAnsatt }: Props) => {
   const intl = useIntl();
+  const { søkInfotrygd } = useFagsakApi();
   const { addErrorMessage, removeErrorMessages } = useRestApiErrorDispatcher();
 
   const [erLosTilgjengelig, setLosErTilgjengelig] = useState(true);
-  const setLosErIkkeTilgjengelig = useCallback(() => {
+  const setLosErIkkeTilgjengelig = () => {
     setLosErTilgjengelig(false);
-  }, []);
+  };
 
   useEffect(() => {
     if (!erLosTilgjengelig) {
```

Comment:
hmm, denne måten å sette errors på virker litt tungvindt outdated? Men må kanskje fortsette med den til alt er over på react-query? For istedenfor å sette en sånn state utifra queries.

`useRestApiError();` kan inspiere cachen å si ifra om det er api errors derfra. Det er kanskje planen?